webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcProtocolHandler.java

```diff
@@ -126,12 +125,9 @@ private record MethodMetrics(Counter callStarted,
                         Http2StreamState currentStreamState,
                         GrpcRouteHandler<REQ, RES> route,
                         GrpcConfig grpcConfig) {
-        this.prologue = prologue;
         this.headers = headers;
         this.streamWriter = streamWriter;
         this.streamId = streamId;
-        this.serverSettings = serverSettings;
-        this.clientSettings = clientSettings;
```

Comment:
It looks as if three ctor parameters--`prologue, serverSettings,` and `clientSettings`--are no longer used. Can they be removed from the parameter list?

Reply:
We certainly can, it always seemed like objects we should be using but never did.